plugins/micron/micron-nvme.c

```diff
@@ -56,6 +56,7 @@ enum eDriveModel {
 	M51CX,
 	M5407,
 	M5411,
+  M6003,
```

Comment:
The indention is off.

Reply:
plugins/micron: fixed the indention issue and update the PR signed off by Inba Vinayagam